django/views/debug.py

```diff
@@ -24,7 +24,7 @@
     libraries={'i18n': 'django.templatetags.i18n'},
 )
 
-HIDDEN_SETTINGS = re.compile('API|TOKEN|KEY|SECRET|PASS|SIGNATURE', flags=re.IGNORECASE)
+HIDDEN_SETTINGS = re.compile('API|TOKEN|KEY|SECRET|PASS|SIGNATURE|URL|DSN', flags=re.IGNORECASE)
```

Comment:
`URL` seems a bit aggressive to me. I can imagine false positives will occur that would hide an interesting setting. For sensitive data, I agree with being better safe than sorry, but this one strikes me as too common. What do others think?